telegram/_bot.py

```diff
@@ -5627,33 +5752,76 @@ async def create_new_sticker_set(
                     File paths as input is also accepted for bots *not* running in
                     :paramref:`~telegram.Bot.local_mode`.
 
-            emojis (:obj:`str`): One or more emoji corresponding to the sticker.
+                .. deprecated:: NEXT.VERSION
+                    |api6_6_depr|
+
+            emojis (:obj:`str`, optional): One or more emoji corresponding to the sticker.
+
+                .. deprecated:: NEXT.VERSION
+                    |api6_6_depr|
+
             mask_position (:class:`telegram.MaskPosition`, optional): Position where the mask
                 should be placed on faces.
-            sticker_type (:obj:`str`, optional): Type of stickers in the set, pass
-                :attr:`telegram.Sticker.REGULAR` or :attr:`telegram.Sticker.MASK`. Custom emoji
-                sticker sets can't be created via the Bot API at the moment. By default, a
-                regular sticker set is created.
 
-                .. versionadded:: 20.0
+                .. deprecated:: NEXT.VERSION
+                    |api6_6_depr|
 
         Returns:
             :obj:`bool`: On success, :obj:`True` is returned.
 
         Raises:
-            :class:`telegram.error.TelegramError`
+            :exc:`TypeError`: Raised when: 1) ``stickers`` and ``sticker_format`` are passed
+                  together with the deprecated parameters. 2) If neither the new parameters or
+                  the deprecated parameters are passed.
 
+            :class:`telegram.error.TelegramError`: For other errors.
         """
+        pre_api_6_6_params = {
+            "emojis": emojis,
+            "png_sticker": png_sticker,
+            "mask_position": mask_position,
+            "tgs_sticker": tgs_sticker,
+            "webm_sticker": webm_sticker,
+        }
+
+        if not any(pre_api_6_6_params.values()) and not all((stickers, sticker_format)):
+            raise TypeError(
+                "Since Bot API 6.6, the parameters `stickers` and `sticker_format` "
+                "are required, please pass them as well."
+            )
+
+        if any(pre_api_6_6_params.values()) and any((stickers, sticker_format)):
+            raise TypeError(
+                "Since Bot API 6.6, the parameters `stickers` and `sticker_format` "
+                "are mutually exclusive with the deprecated parameters "
+                f"{set(pre_api_6_6_params)}. Please use the new parameter "
+                "`stickers` and `sticker_format` instead."
+            )
+            # If we had allowed this, the created sticker set would have used the newer parameters
+            # only, which would have been confusing.
```

Comment:
I don't really get what this comment is about …

Reply:
I passed both older and newer parameters at the same time to Telegram and found that the API will only respect the newer parameters and disregard the older ones

Reply:
okay. I guess it was just natural enough for me to make these things mutually exclusive anyways :D